<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>102</y>
     <width>145</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>61</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>166</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>请输入密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_admin">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>66</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_config">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>66</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>售后</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changePW">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>212</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_login</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login()</slot>
 </slots>
</ui>
